apache-rat-core/src/main/java/org/apache/rat/Report.java

```diff
@@ -349,29 +357,20 @@ private Report() {
      * @return the IReportale instance containing the files.
      */
     private static IReportable getDirectory(String baseDirectory, ReportConfiguration config) {
-        try (PrintStream out = new PrintStream(config.getOutput().get())) {
-            File base = new File(baseDirectory);
-            if (!base.exists()) {
-                out.print("ERROR: ");
-                out.print(baseDirectory);
-                out.print(" does not exist.\n");
-                return null;
-            }
+        File base = new File(baseDirectory);
```

Comment:
FYI, `getDirectory()` method doesn't cleanly close the `Writer` from config, e.g. we have a bug: nothing is printed to `System.out`.
This change here fix the issue.

Reply:
Catcha. Let me rebase and update accordingly.

Reply:
@Claudenw is it already merged on `master` ? I tested on `master` and, even if it should with `out` is `null`, and nothing is printed on `System.out` using CLI. If I remove the lines in this change, it works. So I'm pretty sure that this part is wrong and the `Writer` is not closed cleanly.

For instance:

```
$ java -jar apache-rat-0.16-SNAPSHOT.jar /Users/jbonofre/mytest
$
```

So nothing is printed on `System.out` whereas it shoul.

It's a bug for me, fixed with my change.

Reply:
@jbonofre I merged other changes concerning RAT-321. Pls retry.

Reply:
I confirm that it works now on `master`. I will rebase and clean this PR accordingly.